packages/itwin/property-grid/src/components/PropertyGrid.tsx

```diff
@@ -171,8 +172,37 @@ export const PropertyGrid = ({
           await customOnDataChanged(dataProvider);
         }
       }
+      if(persistNullValueToggle) {
+        let res = await getShowNullValues();
+        if(res !== undefined && dataProvider)
+        {
+          dataProvider.includeFieldsWithNoValues = res;
+          setShowNullValues(res);
+        }        
```

Comment:
could you run prettier against this, there's formatting issues in several places like unexpected trailing whitespace and no space in between the `if` and `(`

Reply:
Ah gotcha, yeah I always miss that stuff since it's not part of the build or commit lol. I'll run it and push fixes now.

Reply:
All current changes have been checked for lint errors and fixes for those are pushed.